src/app/pages/datasets/components/dataset-capacity-management-card/dataset-capacity-management-card.component.html

```diff
@@ -64,9 +64,16 @@ <h3 mat-card-title>
       <div class="details-item">
         <span class="label">{{ 'User Quotas' | translate }}:</span>
         <ng-container *ngIf="checkQuotas; else noneTemplate">
-          <span class="value" *ngIf="!isLoadingQuotas; else valuePlaceholder">
-            {{ 'Quotas set for {n} users' | translate: { n: userQuotas } }}
-          </span>
+          <div *ngIf="!isLoadingQuotas; else valuePlaceholder">
+            <span class="value" *ngIf="userQuotas === 1; else manyQuotas">
```

Comment:
`ICU message format` means a different approach - https://github.com/truenas/webui#translating-text-to-other-languages

Reply:
Done